### ISSUE TITLE
cinder segmentation fault issue 

### ISSUE THREAD

**Issue by StanOnix** [lorin/devstack-vm#23]
I've an issue concerning cinder on line 458. I can't figure out the solution for this. Anyone can help ? 
`==> default: + is_service_enabled c-api
==> default: ++ set +o
==> default: ++ grep xtrace
==> default: + local 'xtrace=set -o xtrace'
==> default: + set +o xtrace
==> default: + return 0
==> default: + is_service_enabled tls-proxy
==> default: ++ grep xtrace
==> default: ++ set +o
==> default: + local 'xtrace=set -o xtrace'
==> default: + set +o xtrace
==> default: + return 1
==> default: + create_volume_types
==> default: + is_service_enabled c-api
==> default: ++ set +o
==> default: ++ grep xtrace
==> default: + local 'xtrace=set -o xtrace'
==> default: + set +o xtrace
==> default: + return 0
==> default: + [[ -n lvm:lvmdriver-1 ]]
==> default: + local be be_name be_type
==> default: + for be in '${CINDER_ENABLED_BACKENDS//,/ }'
==> default: + be_type=lvm
==> default: + be_name=lvmdriver-1
==> default: + cinder type-create lvmdriver-1
==> default: /home/vagrant/devstack/lib/cinder: line 458: 20727 Segmentation fault      (core dumped) cinder type-create ${be_name}
==> default: ++ err_trap
==> default: ++ local r=139
==> default: ++ set +o xtrace
==> default: Error on exit
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
`

**Comment by lorin** [comment]
I am not able to reproduce the issue on my local machine. Can you confirm it fails each time if you start over from scratch?

**Comment by StanOnix** [comment]
Yes, it failed each time I started over from scratch. Do I need to install some specific package about LVM ?

**Comment by lorin** [comment]
You shouldn't need to install additional packages? Can you try updating your Vagrant box? In the devstack-vm directory, do: 

```
$ vagrant box update
```

I had success with v20160406.0.0.

**Comment by StanOnix** [comment]
I have the same version (v20160406.0.0) but even after a fresh restart I have the same problem. I have tested the installation on a collegue's computer but we have the same issue.
I  already try the installation on my personal computer and it worked well even if I need to do a ./stack.sh after every boot :/ . But, I really don't understand why it doesn't work on my company's one.

**Comment by lorin** [comment]
Unfortunately, I have no idea why this would fail repeatedly on a particular type of machine.
